exercises/practice/accumulate/.docs/instructions.md

```diff
@@ -0,0 +1,27 @@
+# Description
+
+Implement the `accumulate` operation, which, given a collection and an
+operation to perform on each element of the collection, returns a new
+collection containing the result of applying that operation to each element of
+the input collection.
+
+Given the collection of numbers:
+
+- 1, 2, 3, 4, 5
+
+And the operation:
+
+- square a number (`[ sq ]`)
+
+Your code should be able to produce the collection of squares:
+
+- 1, 4, 9, 16, 25
+
+Check out the test suite to see the expected function signature.
```

Comment:
This line is a copy-paste error (Thanks @kotp for this)

The "function signature" ([stack effect declaration](https://docs.factorcode.org/content/article-effects.html)) is already provided in `accumulate.factor`